.github/actions/setup-rust/action.yml

```
@@ -2,6 +2,10 @@
 name: "Setup Rust"
 description: "Sets up the correct Rust version and caching via sccache and a GCP backend"
 inputs:
+  key:
+    description: "The key to use for the cache"
+    required: false
+    default: ${{ runner.os }}-${{ runner.arch }}
```

```suggestion
    # e.g. Linux-x86_64
    default: ${{ runner.os }}-${{ runner.arch }}
```

I forgot that `runner.os` doesn't have the whole version like "ubuntu-22.04" in `runs-on`. This would help me remember